apps/www/components/Marketing/index.js

```diff
@@ -53,10 +53,7 @@ const Marketing = ({
       <ColorContextProvider colorSchemeKey='dark'>
         <Top carouselData={data.carousel} />
       </ColorContextProvider>
-      {/*
-        TODO: add on 09.01.2023
-        <ClimateLabTeaser />
-      */}
+      <ClimateLabTeaser />
```

Comment:
We'll ship landing page and climate teaser on marketing front separately. First landing page, climate lab teaser will be shipped after newsletter is sent (evening today or tomorrow in the morning